tree/dataframe/inc/ROOT/RDataFrame.hxx

```diff
@@ -0,0 +1,88 @@
+// Author: Enrico Guiraud, Danilo Piparo CERN  12/2016
+
+/*************************************************************************
+ * Copyright (C) 1995-2016, Rene Brun and Fons Rademakers.               *
+ * All rights reserved.                                                  *
+ *                                                                       *
+ * For the licensing terms see $ROOTSYS/LICENSE.                         *
+ * For the list of contributors see $ROOTSYS/README/CREDITS.             *
+ *************************************************************************/
+
+/**
+  \defgroup dataframe DataFrame
+ROOT's RDataFrame allows to analyse data stored in TTrees with a high level interface.
+*/
+
+#ifndef ROOT_RDATAFRAME
+#define ROOT_RDATAFRAME
+
+#include <iosfwd> // std::ostringstream
```

Comment:
That should need `<sstream>`? fwd is not enough, I believe.

Reply:
A genuine one which never caused issues because of indirect inclusions...